api/app/controllers/api/v1/positions_controller.rb

```diff
@@ -24,6 +29,7 @@ def create
                 render_error(position.errors.full_messages.join("; "))
                 return
             end
+            update_instructors_ids(position)
```

Comment:
Don't you have to save once you've updated the instructor ids for the position?

Reply:
That is actually a very interesting point. I'll check if save affects anything.

Reply:
While it doesn't seem like saving changes the persistence in the db here, should I `save` anyways?

Reply:
I guess we couuuuuuuuuuuuuuuuuuuuuuuld. We'll have to test it somehow. I feel that something will pop up in prod when we try LUL